synapse/lib/cmdr.py

```diff
@@ -40,6 +43,13 @@ async def getItemCmdr(cell, outp=None, color=False, **opts):
         s_cli.Cli: A Cli instance with Cmds loaeded into it.
 
     '''
+    mesg = s_common.deprecated('cmdr', curv='2.164.0')
```

Comment:
`synapse.tools.cmdr`

Reply:
Yeah I considered that but this doesn't only apply to the cmdr tool